### PR TITLE
UX: Added tooltips to topic admin menu

### PR DESCRIPTION
**Pull request by venarius**
Added tooltips to the topic admin menu.

Resolves: https://meta.discourse.org/t/tooltips-for-topic-admin-actions/96086

### CODE REVIEW

File: config/locales/client.en.yml [hunk 1]
```diff
@@ -1924,21 +1924,36 @@ en:
...
         recover: "Un-Delete Topic"
+        recover_tooltip: "Restores the topic."
         delete: "Delete Topic"
+        delete_tooltip: "Marks the topic as deleted."
         open: "Open Topic"
+        open_tooltip: "Marks the topic as open and allows new replies."
         close: "Close Topic"
+        close_tooltip: "Marks the topic as closed and prevents new replies."
         multi_select: "Select Posts…"
+        multi_select_tooltip: "Toggles multi select mode in which multiple posts can be edited."
```
Comment (tgxworld) [inline config/locales/client.en.yml]:
```suggestion
        multi_select_tooltip: "Toggles multi select mode in which multiple posts can be selected."
```

**Comment by tgxworld** [comment]
The code looks good to me. @coding-horror Do you have any thoughts about the copy? It can feel alittle verbose for some.

File: config/locales/client.en.yml [hunk 2]
```diff
@@ -1924,21 +1924,36 @@ en:
...
         recover: "Un-Delete Topic"
+        recover_tooltip: "Restores the topic."
         delete: "Delete Topic"
+        delete_tooltip: "Marks the topic as deleted."
         open: "Open Topic"
+        open_tooltip: "Marks the topic as open and allows new replies."
         close: "Close Topic"
+        close_tooltip: "Marks the topic as closed and prevents new replies."
         multi_select: "Select Posts…"
+        multi_select_tooltip: "Toggles multiselect mode in which multiple posts can be selected."
```
Comment (ZogStriP) [inline config/locales/client.en.yml]: I don't think we ever mention this as the "multiselect mode" anywhere else.

Comment (venarius) [reply config/locales/client.en.yml]: I just took the same name the locale variable has. But I think, "multiselect mode" is pretty easy to understand :) Other ideas on how to call it?

**Comment by SamSaffron** [comment]
I am merging this for now, @coding-horror  can follow up to improve copy! thanks @venarius !

**Comment by coding-horror** [comment]
I don't think this should be merged @sam it's a LOT of extra copy that has to be translated and most of it adds little value over the words that are already there... My vote is that this gets rolled back.

**Comment by SamSaffron** [comment]
I will roll back for now ... but some of the tooltips are handy

1. What does "archive" do? hover on archive to figure out
2. What does "reset bump" do? 
3. What does "unlisting" do? 

I think the failure here is that the change was too far reaching.

**Comment by pgorod** [comment]
I'm on my honeymoon with Discourse, I'm a new admin, and I went looking for this on the Feature Requests, and it brought me here.

It's true that these tooltips have to be translated and that they will rarely be seen, but I don't feel it's true that they don't bring much value. 

My rationale is this: Topic admin actions are things with consequences - you can annoy users, you can make content disappear, you can generate email notifications. I feel the need for extra security before clicking these. Some of these concepts are new to me.

I would vote (recommend? ask?) that this (or something like this) actually gets into the product. I would focus the copy on a logic of "explain consequences, if any, especially if they are irreversible or impact others; provide security to a noobie admin operating this menu".

**Comment by ZogStriP** [comment]
@pgorod if you want to talk about Discourse, highly recommended that you head over https://meta.discourse.org 😉

**Comment by pgorod** [comment]
Ok. But that's where I came from :-) I searched "Feature requests", then I found [somebody had already asked for it](https://meta.discourse.org/t/tooltips-for-topic-admin-actions/96086), then there was [a PR](https://github.com/discourse/discourse/pull/7127) (closed) then there was this PR (reverted). I thought it was better to post here because the latest developments are only here, not on those other places. 

But ok, I get the nudge, I will explain he latest developments on that Meta thread and post there. Thanks.

**Pull request merged**